packages/components/src/horizontal-bar-list/horizontal-bar-grid-item.tsx

```diff
@@ -0,0 +1,68 @@
+import { decodeEntities } from '@wordpress/html-entities';
+import classnames from 'classnames';
+import React from 'react';
+import type { HorizontalBarListItemProps } from './types';
+
+import './style.scss';
+
+const BASE_CLASS_NAME = 'horizontal-bar-list';
+
+const HorizontalBarListItem = ( {
+	data,
+	maxValue,
+	url,
+	onClick,
+	hasIndicator,
+	leftSideItem,
+	rightSideItem,
```

Comment:
I'd like us to clarify this property name — it's not obvious that the "rightSideItem" is only visible on hover.

Reply:
That's a very good point. I'll have to make that optional because `Posts & pages` use this area for hover actions but `Referrers` needs it visible the whole time.